script/install-repos.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+git clone git@github.com:department-of-veterans-affairs/vets-website.git
```

Comment:
Wouldn't this one already be cloned if you were running this script?

Reply:
Yes, that was a late night copy-paste error :)